resources/views/components/content-card.blade.php

```diff
@@ -49,7 +49,13 @@ class="my-1 mr-4 text-sm tracking-wide text-gray-700"
             </div>
         @endif
 
-        <h1 class="mb-4 font-semibold">{{ $model->title }}</h1>
+        @if ($readMore)
+            <a href="{{ route($route, ['slug' => $model->slug]) }}">
+                <h1 class="mb-4 font-semibold">{{ $model->title }}</h1>
+            </a>
+        @else
+            <h1 class="mb-4 font-semibold">{{ $model->title }}</h1>
+        @endif
```

Comment:
Thanks for your contribution!

I completely agree this needs to exists, but I would rather do it like this to avoid repetition. This also adds visual cues that the title is actually a link.

```suggestion
        <h1 class="mb-4 font-semibold">
            @if ($readMore)
                <a href="{{ route($route, ['slug' => $model->slug]) }}" class="inline-block underline hover:no-underline">
                    {{ $model->title }}
                </a>
            @else
                {{ $model->title }}
            @endif
        </h1>
```

Reply:
Fair enough, thanks for the suggestion